README.md

```diff
@@ -30,6 +30,19 @@ npx -p @govtechsg/open-attestation-cli open-attestation <arguments>
 
 ---
 
+## Supported networks
+
+| Network        | Chain ID | Type       |
+| -------------- | -------- | ---------- |
+| mainnet        | 1        | Production |
+| ropsten        | 3        | Test       |
+| rinkeby        | 4        | Test       |
+| goerli         | 5        | Test       |
+| polygon        | 137      | Production |
+| polygon_mumbai | 80001    | Test       |
```

Comment:
Will have to change the names for these 2 as well.

```suggestion
| matic        | 137      | Production |
| maticmum | 80001    | Test       |
```

Reply:
ok, reference from [here](https://docs.ethers.io/v5/api/providers/api-providers/#InfuraProvider).